src/status_im2/contexts/onboarding/sign_in/view.cljs

```diff
@@ -1,42 +1,223 @@
 (ns status-im2.contexts.onboarding.sign-in.view
-  (:require [quo2.core :as quo]
+  (:require ["react-native-camera-kit" :refer (CameraKitCamera)]
+            [clojure.string :as string]
+            [oops.core :as oops]
+            [quo2.core :as quo]
             [quo2.foundations.colors :as colors]
+            [react-native.blur :as blur]
             [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [react-native.permissions :as permissions]
+            [react-native.platform :as platform]
+            [react-native.safe-area :as safe-area]
+            [reagent.core :as reagent]
+            [status-im2.common.resources :as resources]
+            [status-im2.constants :as constants]
             [status-im2.contexts.onboarding.sign-in.style :as style]
             [utils.i18n :as i18n]
-            [status-im2.contexts.onboarding.common.background.view :as background]
             [utils.re-frame :as rf]))
 
-(defn navigation-bar
-  []
-  [rn/view {:style style/navigation-bar}
-   [quo/page-nav
-    {:align-mid?            true
-     :mid-section           {:type :text-only :main-text ""}
-     :left-section          {:type                :blur-bg
-                             :icon                :i/arrow-left
-                             :icon-override-theme :dark
-                             :on-press            #(rf/dispatch [:navigate-back])}
-     :right-section-buttons [{:type                :blur-bg
-                              :icon                :i/info
-                              :icon-override-theme :dark
-                              :on-press            #(js/alert "Pending")}]}]])
-
-(defn page
+(def camera (reagent/adapt-react-class CameraKitCamera))
```

Comment:
This looks like something we should isolate in the `react_native` directory. Maybe `react_native/core.cljs` or a new namespace named `react-native.camera-kit`.

Reply:
Moved 👍